Namco/pac_man.py

```diff
@@ -0,0 +1,28 @@
+class PacMan:
+    def __init__(self, init_row: int = 6, init_col: int = 6):
+        """
+        Implements PacMan
+        """
+        self.row = init_row
+        self.col = init_col
+
+    def process_action(self, action: str) -> [int, int]:
```

Comment:
Itt nem lesz olyan feedback megoldás, hogy ha ezeken kívül más inputot adott meg a felhasználó akkor tájékoztatja őt, hogy az nem jó?

Reply:
Hozzáadtam

Reply:
Akkor jó lesz ez így, az csak extra lenne amúgy is.